ProjectLighthouse.Servers.Website/Pages/Moderation/CasePage.cshtml.cs

```diff
@@ -31,18 +32,27 @@ public async Task<IActionResult> OnGet([FromRoute] int pageNumber, [FromQuery] s
 
         this.SearchValue = name.Replace(" ", string.Empty);
 
-        this.Cases = await this.Database.Cases
-            .Include(c => c.Creator)
-            .Include(c => c.Dismisser)
-            .OrderByDescending(c => c.CaseId)
-            .ToListAsync();
-        
-        this.CaseCount = await this.Database.Cases.CountAsync(c => c.Reason.Contains(this.SearchValue));
-        this.DismissedCaseCount = await this.Database.Cases.CountAsync(c => c.Reason.Contains(this.SearchValue) && c.DismissedAt != null);
+        this.CaseCount =
+            await this.Database.Cases.CountAsync(c => c.Reason.Contains(this.SearchValue));
+        this.ExpiredCaseCount =
+            await this.Database.Cases.CountAsync(c => c.Reason.Contains(this.SearchValue) && c.DismissedAt == null && c.ExpiresAt < DateTime.UtcNow);
+        this.DismissedCaseCount =
+            await this.Database.Cases.CountAsync(c => c.Reason.Contains(this.SearchValue) && c.DismissedAt != null);
```

Comment:
Why does this use the case reason rather than like the person who the case is against? Or this could potentially search against multiple fields like the slot search.

Reply:
Typo, most likely. I'll correct it.